classes/UpgradeTools/CoreUpgrader/CoreUpgrader17.php

```diff
@@ -112,4 +118,58 @@ protected function upgradeLanguage($lang)
             $cldrUpdate->fetchLocale(\Language::getLocaleByIso($isoCode));
         }
     }
+
+    protected function updateTheme()
+    {
+        parent::updateTheme();
+
+        $this->updateRTLFiles();
+    }
+
+    protected function updateRTLFiles()
+    {
+        if (!class_exists(AdaptThemeToRTLLanguagesCommand::class)) {
```

Comment:
I think this will fail if the class doesn't exist?
```suggestion
        if (!class_exists('PrestaShop\PrestaShop\Core\Domain\Theme\Command\AdaptThemeToRTLLanguagesCommand')) {
```

Reply:
No, I've tested this and this work!

Reply:
I confirm it works